modules/ROOT/partials/misc/url-handling.adoc

```diff
@@ -13,7 +13,7 @@ Example: `+http://www.example.com/path1/path2/file.htm+` >> `path2/file.htm`
 
 === Absolute URLs
 
-This will convert all relative URLs to absolute URLs. The URLs will be absolute based on the <<document_base_url,document_base_url>>.
+This will convert all relative URLs to absolute URLs. The URLs will be absolute based on the xref:document_base_url[document_base_url].
```

Comment:
```suggestion
This will convert all relative URLs to absolute URLs. The URLs will be absolute based on the xref:configure/url-handling.adoc#document_base_url[document_base_url].
```

Reply:
done